compiler/src/yul/mappers/expressions.rs

```diff
@@ -272,6 +277,17 @@ fn expr_bool(exp: &Spanned<fe::Expr>) -> Result<yul::Expression, CompileError> {
     unreachable!()
 }
 
+fn expr_str(exp: &Spanned<fe::Expr>) -> Result<yul::Expression, CompileError> {
+    if let fe::Expr::Str(val) = &exp.node {
+        let offset = expression! { dataoffset("foo") };
+        let size = expression! { datasize("foo") };
```

Comment:
@g-r-a-n-t  I have these hardcoded here but they correspond to this line in the YUL

>data \"foo\" \"bar\" 

This seems to be working because I was seeing an error message when I tried to refer to `dataoffset("foo")` but the data point wasn't in the YUL and now that it is in the YUL I'm not seeing that error anymore. So it looks like these should be worken (even if hardcoded for now)